folio-spring-base/src/test/java/org/folio/spring/utils/FolioExecutionContextUtilsTest.java

```diff
@@ -0,0 +1,56 @@
+package org.folio.spring.utils;
+
+import static java.util.Collections.singleton;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.folio.spring.integration.XOkapiHeaders.TENANT;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Consumer;
+import org.folio.spring.DefaultFolioExecutionContext;
+import org.folio.spring.FolioExecutionContext;
+import org.folio.spring.FolioModuleMetadata;
+import org.junit.jupiter.api.Test;
+
+class FolioExecutionContextUtilsTest {
```

Comment:
Add `UnitTest` annotation to have a coverage

Reply:
Fixed, thank you, @psmagin!